client/src/components/map/openlayers/OLMap.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from "react";
+import "./styles.css";
+
+import {Map, layer, Layers} from "react-openlayers";
+
+class OLMap extends Component {
+  render() {
+    return (
+      <div className="Map">
```

Comment:
We use react-bootstrap. You need Container.

Reply:
Thank you for hint. Already fixed. Check it, please.